examples/join-multiple-streams/join_multiple_streams.bal

```diff
@@ -69,7 +69,8 @@ listener http:Listener productMaterialListener = new (9090);
 service productMaterialService on productMaterialListener {
 
     // Initializes the function, which contains streaming queries.
-    future<()> ftr = start initRealtimeProductionAlert();
+    // TODO: issue #17267
+    () ftr = initRealtimeProductionAlert();
```

Comment:
Can't we avoid this.?

Reply:
There is no way to initialize a streams query in a service. What if we initialize it outside.. in package level may be?